src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -4577,37 +4577,35 @@ void CalcPredictedHumidityRatio(EnergyPlusData &state, int const ZoneNum, Real64
     bool SingleSetPoint;                // This determines whether both setpoint are equal or not
     int RoomAirNode;
 
-    auto &Zone = state.dataHeatBal->Zone;
-    auto &HumidityControlZone = state.dataZoneCtrls->HumidityControlZone;
-    auto &RoomAirflowNetworkZoneInfo = state.dataRoomAirMod->RoomAirflowNetworkZoneInfo;
-    auto &ZT = state.dataHeatBalFanSys->ZT;
-    auto &MAT = state.dataHeatBalFanSys->MAT;
-    auto &ZoneAirHumRat = state.dataHeatBalFanSys->ZoneAirHumRat;
-    auto &ZoneAirSolutionAlgo = state.dataHeatBal->ZoneAirSolutionAlgo;
-    auto &ZoneSysMoistureDemand = state.dataZoneEnergyDemand->ZoneSysMoistureDemand;
+    auto &zone = state.dataHeatBal->Zone(ZoneNum);
+    auto &ZT = state.dataHeatBalFanSys->ZT(ZoneNum);
```

Comment:
This may be more cleanup than you want to do in this PR, but at some point you may want to transition all of the zone-level variables in DataHeatBalFanSys from 76 (not a made-up number) arrays of scalars to 1-5 arrays of `struct` (grouped by variable affinity). I know that I may have previously implied that arrays of scalars are superior to arrays of `struct` but that is context/usage-pattern specific and also depends on the size of the struct. In this specific case, I think array management overhead is dominant.